src/install/java/download/gradle.mdx

```diff
@@ -10,34 +10,51 @@ When you use Gradle to download and unzip the `newrelic-java.zip` file, here's w
 * [`newrelic-api.jar`](/docs/agents/java-agent/api-guides/guide-using-java-agent-api#api) (Java agent API jar)
 * README
 
-    Here's how to configure Gradle to download and install the APM agent:
-
-1. Configure your `build.gradle` file with the plugin you will use to download the agent. For example:
-
-    ```
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="step1"
+    title="1. Configure `build.gradle`"
+  >
+Add the following snippet to your `build.gradle` file with the plugin you'll use to download the agent. For example:
+
+    ```java
     plugins {
         id "de.undercouch.download" version "3.4.3"
```

Comment:
The latest version of this plugin is `5.3.0`, so someone should test this method with the new version.

Reply:
Hi @jeffalder. Thanks for giving this update! Is there anyone I should reach out to to test this method with the most recent version?